From: Lubomir Rintel <lkundrak@v3.sk>
Date: Thu, 11 Jan 2018 11:11:10 +0100
Subject: secret-agent: construct the dbus proxy for async agent with the
 correct bus name

The asynchronous secret agent initialization doesn't work at all due to a
rather silly typo. Oops.

While at it, fix a whitespace error too.

(cherry picked from commit 603daa5b256b8964e0eed8c78a49d445f7944734)
---
 libnm/nm-secret-agent-old.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libnm/nm-secret-agent-old.c b/libnm/nm-secret-agent-old.c
index 76e4238..c9dbb4f 100644
--- a/libnm/nm-secret-agent-old.c
+++ b/libnm/nm-secret-agent-old.c
@@ -1118,7 +1118,7 @@ init_async_got_proxy (GObject *object, GAsyncResult *result, gpointer user_data)
 
 	if (priv->auto_register) {
 		nm_secret_agent_old_register_async (init_data->self, init_data->cancellable,
-		                                init_async_registered, init_data);
+		                                    init_async_registered, init_data);
 	} else
 		init_async_complete (init_data, NULL);
 }
@@ -1139,7 +1139,7 @@ init_async_got_bus (GObject *initable, GAsyncResult *result, gpointer user_data)
 	nmdbus_agent_manager_proxy_new (priv->bus,
 	                                  G_DBUS_PROXY_FLAGS_DO_NOT_LOAD_PROPERTIES
 	                                | G_DBUS_PROXY_FLAGS_DO_NOT_AUTO_START,
-	                                NM_DBUS_INTERFACE_AGENT_MANAGER,
+	                                NM_DBUS_SERVICE,
 	                                NM_DBUS_PATH_AGENT_MANAGER,
 	                                init_data->cancellable,
 	                                init_async_got_proxy, init_data);
